geomesa-cassandra/geomesa-cassandra-datastore/src/main/scala/org/locationtech/geomesa/cassandra/data/CassandraDataStoreFactory.scala

```diff
@@ -33,25 +49,57 @@ class CassandraDataStoreFactory extends AbstractDataStoreFactory {
         .withRetryPolicy(DefaultRetryPolicy.INSTANCE)
         .withLoadBalancingPolicy(new TokenAwarePolicy(DCAwareRoundRobinPolicy.builder().build()))
         .build()
+
     val session = cluster.connect(ks)
-    val catalog = CATALOG.lookUp(map).asInstanceOf[String]
-    new CassandraDataStore(session, cluster.getMetadata.getKeyspace(ks), ns, catalog)
-  }
+    val catalog = CatalogParam.lookUp(params).asInstanceOf[String]
 
-  override def createNewDataStore(map: util.Map[String, Serializable]): DataStore = ???
+    // not used but required for config inheritance
+    val queryThreads = QueryThreadsParam.lookupWithDefault[Int](params)
+    val queryTimeout = GeoMesaDataStoreFactory.queryTimeout(params)
+    val looseBBox = LooseBBoxParam.lookupWithDefault[Boolean](params)
+
+    val cfg = CassandraDataStoreConfig(catalog, generateStats, audit, caching, queryThreads, queryTimeout, looseBBox)
+
+    new CassandraDataStore(session, cfg)
+  }
 
   override def getDisplayName: String = "Cassandra (GeoMesa)"
 
   override def getDescription: String = "GeoMesa Cassandra Data Store"
```

Comment:
[minor] you can delegate these to the constants in the CassandraDataStoreFactory object so that they're standardized (the constants there aren't used for anything else)